### PR TITLE
add resetOption method

### PR DESCRIPTION
**Pull request by qroques**

### CODE REVIEW

**Comment by alexpozzi** [comment]
Thank you for your contribution!

Can you please fix StyleCI errors, add a test case for this new method and add a little documentation on how to use it in the README.md file?

File: README.md [hunk 1]
```diff
@@ -80,6 +80,17 @@ $snappy->setOption('toc', true);
 $snappy->setOption('cache-dir', '/path/to/cache/dir');
 ```
 
+### Reset options
+Options can be reset to their initial values with `resetOption()` method.
```
Comment (alexpozzi) [inline README.md]:
```suggestion
Options can be reset to their initial values with `resetOptions()` method.
```

File: test/Knp/Snappy/AbstractGeneratorTest.php [hunk 2]
```diff
@@ -923,4 +923,26 @@ public function testleanupTemporaryFiles()
...
+            ])
+            ->setConstructorArgs(['the_binary', [], ['PATH' => '/usr/bin']])
+            ->getMock()
+        ;
+
+        $media->setOption('copies', 4);
+        $media->setOption('disable-javascript', true);
+        $media->setOption('no-background', true);
+
+        $media->resetOptions();
```
Comment (alexpozzi) [inline test/Knp/Snappy/AbstractGeneratorTest.php]: Can you assert that options has been set before calling `resetOptions`?

Comment (qroques) [reply test/Knp/Snappy/AbstractGeneratorTest.php]: Setting option is already tested with `testSetOption()` method. I don't know we should test it here.

**Pull request merged**